src/nft/components/collection/FilterButton.tsx

```diff
@@ -66,12 +67,15 @@ export const FilterButton = ({
           )}
 
           <FilterIcon
-            style={{ marginBottom: '-4px', paddingRight: `${!isFiltersExpanded || showFilterBadge ? '6px' : '0px'}` }}
+            style={{
+              marginBottom: '-4px',
+              paddingRight: `${!isFiltersExpanded || showFilterBadge ? '6px' : '0px'}`,
```

Comment:
did this change? or just prettier?

Reply:
changed now, added some new stuff, but previously i think just prettier